src/views/DatasetManager/components/DatasetOverview/components/DatasetOverviewPlaceholder/DatasetOverviewPlaceholder.js

```diff
@@ -94,12 +95,23 @@ export const DatasetOverviewPlaceholder = () => {
     if (
       currentDatasetId == null ||
       !currentDatasetIngestionStatus ||
-      ![INGESTION_STATUS.ERROR, INGESTION_STATUS.UNKNOWN].includes(currentDatasetIngestionStatus)
+      ![INGESTION_STATUS.ERROR, INGESTION_STATUS.UNKNOWN, INGESTION_STATUS.PENDING].includes(
+        currentDatasetIngestionStatus
+      )
     )
       return null;
 
     return currentDatasetType === DATASET_SOURCE_TYPE.LOCAL_FILE ? (
       <ReuploadFileDatasetButton datasetId={currentDatasetId} iconButton={false} />
+    ) : currentDatasetIngestionStatus === INGESTION_STATUS.PENDING ? (
+      <Button
+        data-cy="dataset-overview-abort-button"
+        variant="contained"
+        color="error"
+        onClick={() => stopRunner(currentDatasetId)}
```

Comment:
just a question : here, the hook uses `currentDatasetId` as argument but it isn't used elsewhere in the dispatcher or in the saga; on the other hand, the hook takes as argument `runnerId` that we never passed and then, in the saga, both values are calculated from the state :thinking: I must miss something, could you explain how is it wired ? :slightly_smiling_face:

Reply:
Yes, I should have removed this parameter after having made changes in the saga.